### PR TITLE
Added noddity to blogging platforms

### PR DESCRIPTION
**Pull request by ArtskydJ**
Added Noddity to the list of blogging platforms.
Also fixed a typo in the pelican description; `sight -> site`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,7 +70,8 @@ Table of Contents
   * [Dotclear](http://dotclear.org/) - Take control over your blog! ([Source code](https://hg.dotclear.org/dotclear)) `GPLv2` `PHP`
   * [Ghost](http://ghost.org/) - Just a blogging platform ([Source code](https://github.com/TryGhost/Ghost)) `MIT` `Nodejs`
   * [htmly](http://www.htmly.com/) - Databaseless Blogging Platform (Flat-File Blog) ([Source code]((https://github.com/danpros/htmly)), [Demo](http://www.htmly.com/demo/)) `GPLv2` `PHP`
-  * [Pelican](http://blog.getpelican.com) - Static sight generator written in Python, requires no database or server-side logic ([Source code](https://github.com/getpelican/pelican)) `GPLv3` `Python`
+  * [Noddity](http://noddity.com/) - It's a blog, it's a wiki, it's a fast CMS! ([Source code](https://github.com/TehShrike/noddity)) `WTFPL` `Javascript`
```
Comment (nodiscc) [inline README.md]: Please change `Javascript` to `Nodejs` as it is more informative regarding the platform required to run the software.

Comment (ArtskydJ) [inline README.md]: Noddity is compiled once using browserify (nodejs program), but then can be hosted by a static file server.

Would you still like it changed?

Comment (nodiscc) [inline README.md]: Since Nodejs is a requirement to get it up and running, I think the Nodejs tag applies.

**Comment by nodiscc** [comment]
Also, Pelican should be removed when we fix https://github.com/Kickball/awesome-selfhosted/issues/65 (where is it gone???)

**Comment by ArtskydJ** [comment]
Platform is now `Nodejs`.

**Comment by nodiscc** [comment]
Thanks!

**Comment by ArtskydJ** [comment]
You're welcome; thank you.

**Pull request merged**